cla_backend/apps/cla_butler/tests/test_tasks.py

```diff
@@ -170,4 +173,67 @@ def test_delete_old_data_run_case_under_three_years_unsuccessful_delete(self):
         pks = get_pks(Case.objects.all())
         eods = EODDetails.objects.filter(case_id__in=pks).values_list("pk", flat=True)
 
-        self.data_deletion_check(pks, eods, 1, 1)
+        self.data_deletion_check(pks, eods, 1, 1, self.delete_option_three_years)
+
+    def test_delete_no_personal_details_success(self):
+        """
+        This tests a case with no personal details is removed successfully
+        """
+        log = make_recipe("cla_auditlog.audit_log")
```

Comment:
Can the main body of this three tests be refactored into a common method (e.g. `setup_personal_details_test`)? It may be my eye deceiving me but lines 182-194 seem to be the same in each?

Reply:
Create a `setup_personal_details_test` that set's up the repeated test data